project.clj

```diff
@@ -1,4 +1,4 @@
-(defproject hikari-cp "1.7.5"
+(defproject hikari-cp "1.7.6"
```

Comment:
+1, or even rebase to remove this commit from the branch. That'd be the best solution. Thank you @sbitteker!